src/mainwindow.cpp

```diff
@@ -718,69 +718,95 @@ void MainWindow::askDeleteFile()
         return;
     }
 
-    auto trashString = tr("Are you sure you want to move %1 to the Trash?").arg(fileName);
+    QString messageText;
+    if (permanent)
+    {
+        messageText = tr("Are you sure you want to delete %1 permanently? This can't be undone.").arg(fileName);

```

Comment:
In practice I found having the adverb separated, especially if there's a longer filename filled in for the placeholder, to be a bit awkward. Could consider wording this "permanently delete %1" instead.